Makefile

```diff
@@ -36,7 +36,7 @@ FLAGS+=	--plugin=protoc-gen-grpc=$(GRPCPLUGIN)
 FLAGS+= --grpc_out=$(OUTPUT)
 endif
 
-all: clean stroeer/*
+all: stroeer/*
```

Comment:
removed clean from `all` target, because running `java` and `node` sequentially deletes the generated files from the previous run

Reply:
if we keep on specifying distinct output directories (`OUTPUT=./node`) cleaning should be fine?!

Reply:
if we fix the `grpc_out` for `node` cleaning should be fine

Reply:
but the clean task will remove java files if we run the node generation afterwards or vice versa